pkg/asset/agent/manifests/agentclusterinstall.go

```diff
@@ -231,7 +233,12 @@ func (a *AgentClusterInstall) Generate(_ context.Context, dependencies asset.Par
 		}
 
 		if installConfig.Config.Proxy != nil {
-			agentClusterInstall.Spec.Proxy = (*hiveext.Proxy)(getProxy(installConfig.Config.Proxy))
+			rendezvousIP := ""
+			if agentConfig.Config != nil {
+				rendezvousIP = agentConfig.Config.RendezvousIP
```

Comment:
Since this will end up in the cluster config, there's something to be said for adding the whole Machine CIDR (which assisted will add again anyway), so that no single node is configured differently to the others.